scipy/stats/_qmc.py

```diff
@@ -151,6 +159,9 @@ def discrepancy(sample, iterative=False, method='CD'):
     method : str, optional
         Type of discrepancy, can be ``CD``, ``WD``, ``MD`` or ``L2-star``.
         Refer to the notes for more details. Default is ``CD``.
+    workers : int, optional
+        Number of workers to use for parallel processing. If -1 is given all
+        CPU threads are used. Default: 1.
```

Comment:
Through the documentation, `is` is more used.
```suggestion
        CPU threads are used. Default is 1.
```